### DISCUSSION TITLE
Available window locations 16

### DISCUSSION THREAD

**Discussion by byzuzayli** [KevinBytesTheDust/CVAmp#269] [Q&A]
Hello, although I tried it on 3 different machines, I can open the same number of windows.
I can only open 16 windows on all my machines
![resim_2024-05-19_224238224](https://github.com/jlplenio/crude-twitch-viewer-bot/assets/89015538/fca8e313-960d-4175-83f5-9c4692dca2de)

1. machine
i5 9600K
64gb ddr4 3200 mhz

2. machine Notebook
i7 4700 HQ
8gb

3. Machine
2x e5 - 2680v4 Xeon core 28 threads 56
192gb DDr4 2400mhz ECC

**Comment by jlplenio** [comment]
Did you try in headless mode, or do you need to start in headful mode?

**Comment by byzuzayli** [comment]
yes, headless mode, After opening the 15th window, others gradually fail and close.

**Comment by juanmiromero** [comment]
I have something similar,

I checked on an MSI Prestige Laptop with an Intel Core Ultra 7 155H and 32Gb RAM. I never had more than 16, but on a Ryzen 5600x with 32Gb, the Windows locations increased to 30. So, I don't know what affects the number of Windows locations because the Ryzen is older than the Intel processor.
